hatchet/tests/conftest.py

```diff
@@ -347,6 +347,176 @@ def mock_graph_literal():
     return graph_dict
 
 
+@pytest.fixture
+def mock_graph_literal_inc_time_as_foo():
```

Comment:
IIRC you discovered this issue because your graphframe only had `time (inc)` as an inc metric and no exc metrics, right? I'm thinking we should modify this mock graph to have `time (inc)` as the inc metric and `line` or some other metric to be the exc metric.

Reply:
That's a great point. I'll make this change.

Reply:
Fixed by a1d3fb6